sdlib/dmd/gc.d

```diff
@@ -110,6 +110,34 @@ bool __sd_gc_reserve_array_capacity(void* ptr, size_t request,
 		threadCache.reserve(ptr[0 .. existingUsed + 1], request - existingUsed);
 }
 
+size_t __sd_gc_get_array_capacity(void[] slice) {
```

Comment:
What's wrong with the existing getCapacity implementation?

Reply:
Uh... cause I didn't know it was there.

And looking at the impl. It matches exactly (except for the off by one which is easy to fix).